src/content/RepoPage/RepoPage.js

```diff
@@ -70,4 +137,33 @@ const RepoPage = () => {
   );
 };
 
+<Query query={REPO_QUERY}>
```

Comment:
This should replace the `<RepoTable headers={headers} rows={rows} />` line.

Reply:
Changed Done